mojo/public/cpp/platform/tcp_platform_handle_utils_posix.cc

```diff
@@ -168,4 +158,23 @@ bool TCPServerAcceptConnection(const base::PlatformFile server_socket,
 #endif  // defined(OS_NACL)
 }
 
+COMPONENT_EXPORT(MOJO_CPP_PLATFORM)
+bool IsTcpSocket(const base::ScopedFD& fd) {
+  struct sockaddr_storage addr;
+  socklen_t len = sizeof(addr);
+  if (!getsockname(fd.get(), (struct sockaddr*)&addr, &len)) {
+    return (addr.ss_family == AF_INET);
```

Comment:
udp is also AF_INET

Reply:
Currently, We don't use UDP..  The function purpose is to check if fd is AF_INET or not(UNIX domain socket).
It it OK to change the function name to IsInetSocket()?

Reply:
OK, it's better to change the function name, how about IsNetworkSocket()?

Reply:
done